app/controllers/api/v1/tasks_controller.rb

```diff
@@ -28,13 +29,25 @@ def create
                         error: "Please fill out all required fields"
                     }
                 end
+
+                user = User.new(params[:user])
```

Comment:
we should also be more specific with the specific user here, so these should all be `parent`